wix/README.md

```diff
@@ -5,11 +5,13 @@ The WIX toolset is required to build the Windows Installer file, the WIX toolset
 It can be downloaded from http://wixtoolset.org.
 
 ## Build Instructions
+Update the dependencies.wxs file by running ./generate-dependencies.sh (in mingw64 bash shell).
+
 Run candle.exe to build wixobj file from the wxs file:
-candle.exe -arch x64 windows-connector.wxs
+candle.exe -arch x64 windows-connector.wxs dependencies.wxs
 
 Run light.exe to build MSI file from the wixobj
-light.exe -ext "C:\Program Files (x86)\WiX Toolset v3.10\bin\WixUIExtension.dll" windows-connector.wixobj
+light.exe -ext "C:\Program Files (x86)\WiX Toolset v3.10\bin\WixUIExtension.dll" windows-connector.wixobj dependencies.wixobj -o windows-connector.msi
 
 If the WIX toolset is installed to a different directory, use that directory path for the
 UI extension dll.
```

Comment:
What is the UI extension dll? It would be nice to indicate that the default path to these tools is this:

/c/Program\ Files\ \(x86\)/WiX\ Toolset\ v3.10/bin/light.exe